tests/test_core.py

```diff
@@ -185,11 +185,13 @@ def test_template_context(self):
 class MenuIntegrationTestCase(FlaskTestCase):
 
     def test_without_menu(self):
-        # it must raise an exception because is not registered.
-        self.assertRaises(RuntimeError, Breadcrumbs, self.app)
```

Comment:
I knew about the *args, but I did not tried it, I just went for the safe known option for 2.6. :-)